Formula/xdk.rb

```diff
@@ -0,0 +1,23 @@
+class Xdk < Formula
+  desc "Ecstasy Development Kit (XDK)"
+  homepage "https://github.com/xtclang/xvm/"
+  url "https://github.com/xtclang/xvm/releases/download/v0.4.2/xdk-0.4.2.tar.gz"
+  sha256 "b0b5217abf05de986e6211a7c8665b304fa89df1726749e4f1dd7eff1cbbe1fe"
+  license "Apache-2.0"
+
+  depends_on "openjdk"
+
+  def install
+    libexec.install buildpath.children
+    %w[xec xtc xam].each do |cmd|
+      java = "${JAVA_HOME}/bin/java"
+      args = "-Xms256m -Xmx1024m -ea -jar #{libexec}/javatools/javatools.jar" \
+             " #{cmd} -L #{libexec}/lib -L #{libexec}/javatools"
+      (bin/cmd).write_env_script java, args, Language::Java.overridable_java_home_env
```

Comment:
This can also be done as:
```suggestion
    %w[xec xtc xam].each do |cmd|
      bin.write_jar_script java, libexec/"javatools/javatools.jar", cmd
```

Reply:
Hi @SmillerDev - that's exactly the path I'm learning about and experimenting with now. Thank you.

Hi @chenrui333 - thank you for the advice and the offer. I haven't worked very often with branches before (smaller projects), so I may have some more questions for you when I actually go through the steps that you're describing. I don't want to waste your time, though, so I'll make sure to learn enough to ask more intelligent questions. I know that this stuff is pretty straight-forward after you've been through it, but I'm not to that point yet with Git + GitHub + pull requests.

Reply:
OK @chenrui333, I think I will need some help to avoid messing up git and my pull request 😊

I squashed my work and committed it as a branch:
https://github.com/xtclang/homebrew-core/commit/d22beb4c9835cb852398622e05c615f759b91a89

Is this the correct first step? If so, how do I update my pull request now with that new commit?

Reply:
It should be in this branch to update the pull request.

Reply:
Hi @SMillerDev, I am not sure what you are saying. My original pull request was in my fork's "master" branch. Is that the "this branch" that you are referring to? @chenrui333  was indicating that "_you should use some branch other master for creating the PR_". I guess I just want to avoid causing any more mess by having to re-create another PR. What are you advising?

Reply:
Just to make sure: With the goal of updating this pull request, what branch name in my fork [xtclang/homebrew-core](https://github.com/xtclang/homebrew-core/) should I push my changes to? i.e. "master" or "xdk-new-formula"?